### PR TITLE
added chile to spanish variants

### PR DESCRIPTION
**Pull request by mybooking-es**

### CODE REVIEW

File: r18n-core/lib/r18n-core/locales/es-cl.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require_relative 'es'
+
+module R18n
+  module Locales
+    # Chile Spanish locale
+    class EsCL < Es
+      set title: 'Chile Spanish',
```
[the remark this answers is not in the document]
Comment (mybooking-es) [reply r18n-core/lib/r18n-core/locales/es-cl.rb]: Thank you. It's done.

File: r18n-core/lib/r18n-core/locales/es-cl.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+          sublocales: %w[es]
+    end
```
[the remark this answers is not in the document]
Comment (mybooking-es) [reply r18n-core/lib/r18n-core/locales/es-cl.rb]: Thank you. It's done

**Comment by mybooking-es** [comment]
Hi Alex,

All the suggested changes were made.

Regards

Juan

**Comment by AlexWayfer** [comment]
Thank you for contributing!

**Pull request merged**